### ISSUE TITLE
POB doesnt count more projectiles added to molten strike on the DPS count.

### ISSUE THREAD

**Issue by asdf337** [Openarl/PathOfBuilding#981]
Either melee hit or molten balls increase in dps when I add more molten strike projectiles. Obviously melee hit shouldnt go up, but the molten balls should increase in dmg, or the total damage should go up. Is this a bug? Why doesnt it show damage from adding more projectiles?
Thanks :)

**Comment by Exportforce** [comment]
Damage is per ball. So you gotta do DMG multiplied by the number of balls.
BUT keep in mind that not all will ever hit. So best you can do is damage * balls / 2
This will be roughly a viable number.

**Comment by asdf337** [comment]
Alright, thanks. Ive also noticed that multistrike doesnt show a dps increase. Are all these things possible to implement or will they not be implemented in the future? Just thought I should report it in for u guys :)

**Comment by Exportforce** [comment]
I don't think he will implement this as it is impossible to create a real DPS number as it matters on how many balls hit. If you're surrounded by masses of mobs it MIGHT be possible that all balls hit once. If you're unlucky on a boss you might hit only with one single ball. 

What a nice thing would be is the following:
Count the projectiles (no matter what skill) and offer either the settings to set a % of balls hitting, so we can tell POB "50% of the projectiles hit" for example or offer just 4 premade infos of 100%, 75%, 50%, 25%

**Comment by asdf337** [comment]
Yeah, sounds like a good option. And the multistrike needs to be fixed as
well, it doesnt add dps.

**Comment by Wires77** [comment]
That's because multistrike is only increasing the number of balls again.  Multistrike works for the melee portion of the attack.

**Comment by asdf337** [comment]
There should be an average estimation on the total damage of it, with the
calculations of the average numbers of balls hitting depending on if its
white mobs / standard boss / shaper.

**Comment by bdg0** [comment]
https://github.com/Openarl/PathOfBuilding/issues/241

`This is a limitation that applies to a lot of skills that have multiple parts; in this case, the Magma Balls part is a projectile skill, so its attack speed must be affected by projectile attack modifiers, just as its damage is. The attack/cast rate for secondary skill parts should generally be ignored anyway, or at least taken with a grain of salt, as the skill mechanics often render it inaccurate; it's only shown at all because it is sometimes correct.`

**Comment by asdf337** [comment]
Thanks.
